build_image/docker/agent/kubernetes/Dockerfile.in

```diff
@@ -10,6 +10,6 @@ COPY src/agent/kubernetes-agent/src /app
 WORKDIR /app
 
 ENV KUBECONFIG /app/.kube/config
-ENV PYTHONPATH /app:$PATHONPATH
+ENV PYTHONPATH /app:/usr/local/lib/python3.6:/usr/local/lib/python3.6/lib-dynload:/usr/local/lib/python3.6/site-packages
```

Comment:
It's better get python version through PYTHON_VERSION env, and set here ${YOUR_PYTHON_VERSION}